api/setting.go

```diff
@@ -42,7 +42,7 @@ func (h SettingHandler) AddRoutes(e *gin.Engine) {
 // @description Get a setting by its key.
 // @tags get, setting
 // @produce json
-// @success 200 {object} interface{}
+// @success 200 {object} api.Setting
```

Comment:
```{object}``` or ```interface{}``` are not supported types for the generator, more details: https://swagger.io/docs/specification/data-models/data-types/